src/core/db/rlsmanager.py

```diff
@@ -0,0 +1,73 @@
+from config import settings
+from django.contrib.auth.models import User
+import os
+import core.db.connection
+
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+
+
+class RowLevelSecurityManager:
+
+    def __init__(self, user, table, repo, repo_base):
+        if user != 'dh_public' and user != 'postgres':
+            user = User.objects.get(username=user)
+            self.username = user.username
+        else:
+            self.username = user
+
+        self.repo_base = repo_base
+        self.repo = repo
+        self.table = table
+
+        self.user_con = core.db.connection.DataHubConnection(
```

Comment:
@justinanderson did some major work on how the DataHubManager handles connections and connection pooling. It's the reason that you (now) see things like 

```
with DataHubManager(user=username, repo_base=repo_base) as manager:
        repos = manager.list_repos()```

scattered around the code. It also prevents us from opening up too many connections to the database, and crashing the server. We'll need to upgrade the `RowLevelSecurityManager` before merging, for roughly the same reasons. I or Justin can help you with this when you're ready.